### PR TITLE
Category Management (including Batch Categorization formerly known as Cat-A-Lot)

### PR DESCRIPTION
**Pull request by FlominatorTM**
For faster categorization:
grafik

### CODE REVIEW

File: .vscode/firefox_build_and_run.bat [hunk 1]
```diff
@@ -0,0 +1,4 @@
+%~d0
```
Comment (FlominatorTM) [inline .vscode/firefox_build_and_run.bat]: I will probably still need to remove those two, right?

Comment (shogenapps) [reply .vscode/firefox_build_and_run.bat]: Yes, I think so.  Put them in .gitignore.

Comment (FlominatorTM) [reply .vscode/firefox_build_and_run.bat]: or maybe just the tasks, but keeping the bat?

Comment (FlominatorTM) [reply .vscode/firefox_build_and_run.bat]: managed to put it in package.json, so it should be ok now

**Pull request merged**